core/clock/impl/chain_epoch_clock_impl.cpp

```diff
@@ -16,7 +16,7 @@ namespace fc::clock {
   outcome::result<ChainEpoch> ChainEpochClockImpl::epochAtTime(
       const Time &time) const {
     if (time < genesis_time_) {
-      return EpochAtTimeError::BEFORE_GENESIS;
+      return outcome::failure(EpochAtTimeError::BEFORE_GENESIS);
```

Comment:
why? it used to be cast to failure already.

Reply:
It casts to ChainEpoch now.